docs/topics/logging.rst

```diff
@@ -242,6 +242,48 @@ e.g. in the spider's ``__init__`` method::
 If you run this spider again then INFO messages from
 ``scrapy.spidermiddlewares.httperror`` logger will be gone.
 
+You can also filter messages by content using a substring or
```

Comment:
Without going into specifics in the examples below, as to keep the section short, I would still make references to the `LogRecord` class and the fact that it offers more possibilities for filtering beyond log messages, e.g.

```suggestion
You can also filter log records by :class:`~logging.LogRecord` data. For example, you can filter log records by message content using a substring or
```